uaclient/api/u/pro/unattended_upgrades/status/v1.py

```diff
@@ -0,0 +1,188 @@
+import datetime
+import os
+from typing import Dict, List, Optional, Tuple, Union
+
+from uaclient import exceptions, messages
+from uaclient.api.api import APIEndpoint
+from uaclient.api.data_types import AdditionalInfo
+from uaclient.api.exceptions import UnattendedUpgradesError
+from uaclient.apt import get_apt_config_keys, get_apt_config_values
+from uaclient.config import UAConfig
+from uaclient.data_types import (
+    BoolDataValue,
+    DataObject,
+    DatetimeDataValue,
+    Field,
+    IntDataValue,
+    StringDataValue,
+)
+from uaclient.system import get_systemd_job_state
+
+UNATTENDED_UPGRADES_CONFIG_KEYS = [
+    "APT::Periodic::Enable",
+    "APT::Periodic::Update-Package-Lists",
+    "APT::Periodic::Unattended-Upgrade",
+    "Unattended-Upgrade::Allowed-Origins",
+]
+
+UNATTENDED_UPGRADES_STAMP_PATH = "/var/lib/apt/periodic/upgrade-stamp"
+
+
+class UnattendedUpgradesStatusResult(DataObject, AdditionalInfo):
+    fields = [
+        Field("systemd_apt_timer_enabled", BoolDataValue),
+        Field("apt_periodic_job_enabled", BoolDataValue),
+        Field("package_lists_refresh_frequency_days", IntDataValue),
+        Field("unattended_upgrades_frequency_days", IntDataValue),
+        Field("unattended_upgrades_allowed_origins", StringDataValue),
+        Field("unattended_upgrades_running", BoolDataValue),
+        Field("unattended_upgrades_disabled_reason", StringDataValue),
```

Comment:
I think this may need to be a "named string" (e.g. `{ "code": "message-code", "message": "human readable" }`) so that the consumer can choose or translate the actual message as they see fit

Reply:
That's a good point, let me update this